joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/dataflow/SingleAssignmentTests.scala

```diff
@@ -16,14 +16,12 @@ class SingleAssignmentTests extends RubyCode2CpgFixture(withPostProcessing = tru
     val source = cpg.literal.l
     val sink   = cpg.method.name("puts").callIn.argument.l
     val flows  = sink.reachableByFlows(source).map(flowToResultPairs).distinct.sortBy(_.length).l
-    flows.size shouldBe 6
-    val List(flow1, flow2, flow3, flow4, flow5, flow6) = flows
+    val List(flow1, flow2, flow3, flow4, flow5) = flows
```

Comment:
The previously called `flow3` is removed, but I reckon the new `flow4` covers it better.